src/BootstrapForm.php

```diff
@@ -172,8 +181,12 @@ public function staticField($name, $label = null, $value = null, $options = arra
 
         $label = $this->getLabelTitle($label, $name);
         $inputElement = '<p'.$this->html->attributes($options).'>'.e($value).'</p>';
-
-        $wrapperOptions = ['class' => $this->getRightColumnClass()];
+		
+		$wrapperOptions = [];
+		if($this->use_column_classes){
```

Comment:
There should be a space between `if` and the opening bracket. There should also be a space between the closing bracket and the opening brace. Also, indentation is still off.